classes/PodsI18n.php

```diff
@@ -196,6 +198,10 @@ private static function default_strings() {
 
 			'%s items'                                     => __( '%s items', 'pods' ),
 
+			'You can only select %s item'                  => __( 'You can only select %s item', 'pods' ),
+
+			'You can only select %s items'                 => __( 'You can only select %s items', 'pods' ),
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: efe013e31dd631fb29594c9cf565b54c</sub>